### ISSUE TITLE
Screenshot aborts after using text and then selecting any other tool

### ISSUE THREAD

**Issue by nikola-ffw** [flameshot-org/flameshot#3265]
### Flameshot Version

Flameshot v11.0.0 (11.0.0-2 Debian)
Compiled with Qt 5.15.2
linux: 5.19.0-46-generic
ubuntu: 22.04

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Ubuntu 22.04.2 LTS (64-bit)

### Description

After using the "text" tool type something, selecting any other tool like "line", "arrow" etc., causes the screenshot to get aborted.
The screenshot window disappears and "Screnshot aborted" message is displayed.

### Steps to reproduce

1. run the GUI
2. start creating a screenshot
3. use text tool to add some copy
4. try using any other tool after that (line, arrow, rectangle...)

### System Information

# Ubuntu 22.04.2 LTS (64-bit)
![image](https://github.com/flameshot-org/flameshot/assets/60426447/1cf9f4bc-a77a-40ac-99b8-96fe96b2df7f)
```
Flameshot v11.0.0 (11.0.0-2 Debian)
Compiled with Qt 5.15.2
```
```
Linux nix-ubuntu 5.19.0-46-generic #47~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Jun 21 15:35:31 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```
```
VGA compatible controller: NVIDIA Corporation TU116 [GeForce GTX 1660 SUPER] (rev a1)
```

**Comment by paulpreibisch** [comment]
I have listed same bug here: https://github.com/flameshot-org/flameshot/issues/3102

**Comment by mmahmoudian** [comment]
These are both duplicates of #2238 (and 9 other duplicates) which have been already resolved by #2369 on 2022-02-05 which is one and a half years a go. You both are using version 11.0.0 and we are now on 12.1.0 for the past year
